lms/djangoapps/api_manager/courses/views.py

```diff
@@ -1539,9 +1539,17 @@ def get(self, request, course_id):  # pylint: disable=W0613
             users_enrolled_qs = users_enrolled_qs.filter(organizations=organization)
             org_ids = [organization]
 
-        users_started = StudentProgress.get_num_users_started(course_key, exclude_users=exclude_users, org_ids=org_ids)
+        group_ids = self.request.QUERY_PARAMS.get('groups', None)
+        if group_ids:
+            group_ids = map(int, group_ids.split(','))
```

Comment:
@msaqib52 what will happend if we get a bad value in groups param like `groups=bad_value`?